src/state-summary/state-card-timer.html

```diff
@@ -0,0 +1,88 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+
+<link rel="import" href="../../bower_components/iron-flex-layout/iron-flex-layout-classes.html">
+
+<link rel="import" href="../components/entity/state-info.html">
+<link rel="import" href="../util/hass-mixins.html">
+<link rel="import" href="../util/hass-util.html">
+
+<dom-module id="state-card-timer">
+  <template>
+    <style is="custom-style" include="iron-flex iron-flex-alignment"></style>
+    <style>
+      .state {
+        @apply(--paper-font-body1);
+        color: var(--primary-text-color);
+
+        margin-left: 16px;
+        text-align: right;
+        line-height: 40px;
+      }
+    </style>
+
+    <div class='horizontal justified layout'>
+      <state-info state-obj="[[stateObj]]" in-dialog='[[inDialog]]'></state-info>
+      <div class='state'>[[secondsToDuration(timeRemaining)]] <i>([[stateObj.state]])</i></div>
```

Comment:
Just an opinion, but I think the active/inactive text here isn't necessary. It's already pretty clear when a timer is active from the icon and the fact that the time is changing. IMO simpler is better.

If we do want to keep it we should pass it through `computeStateDisplay` for localization.

Reply:
Ok, I'll remove it.